spec.html

```diff
@@ -7598,7 +7598,7 @@
         <emu-alg>
           1. Let _args_ be the arguments object.
           1. If _P_ is `"caller"`, then
-            1. Let _desc_ be ! OrdinaryGetOwnProperty(_args_, _P_).
+            1. Let _desc_ be OrdinaryGetOwnProperty(_args_, _P_).
             1. If IsDataDescriptor(_desc_) is *true*, return *true*.
```

Comment:
I might not understand completion records completely, but OrdinaryGetOwnProperty only ever returns undefined or a Property Descriptor. How can that be abrupt?